snoop/data/analyzers/emlx.py

```diff
@@ -18,13 +20,21 @@ def reconstruct(file_pk):
         if part.get('X-Apple-Content-Length'):
             ext = f'.{ref}.emlxpart'
             part_name = re.sub(r'\.partial\.emlx$', ext, file.name)
+            parent = file.parent_directory
+
+            require_dependency(
+                f'walk-{parent.pk}', depends_on,
+                lambda: filesystem.walk.laterz(parent.pk),
+            )
+
+            require_dependency(
+                f'walk_file-{parent.pk}-part-{ext}', depends_on,
```

Comment:
Instead of `-part-{ext}` maybe put in the actual filename? Or say `-emlxpart` instead of `-part` so it's obvious it's about emlx files?

Reply:
`emlxpart-` works. I'd rather not use the filename, who knows what scary stuff is in there.